versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5775 @@
+* `wf.other_workflow2.foobar.input` - References the `File` input of the call to `foobar` inside the second call to subworkflow `other.other_workflow`
+* `wf.arr` - References the `Array[String]` declaration on the workflow
+* `wf.scattered_test` - References the scattered version of `call test`
+* `wf.scattered_test.my_var` - References an `Array[String]` for each element used as `my_var` when running the scattered version of `call test`.
+* `wf.scattered_test.results` - References an `Array[File]` which are the accumulated results from scattering `call test`
+* `wf.scattered_test.1.results` - References an `File` from the second invocation (0-indexed) of `call test` within the scatter block.  This particular invocation used value "b" for `my_var`
+
+A namespaced identifier has the same syntax as a fully-qualified name. It is interpreted as the left-hand side being the name of a namespace and the right-hand side being the name of a workflow, task, or sub-namespace within that namespace.
+
+Consider this workflow:
+
+```wdl
+import "other.wdl" as ns
+
+workflow wf {
+  call ns.ns2.task
+}
+```
+
+Here, `ns.ns2.task` is a namespace identifier (see the [Call Statement](#call-statement) section for more details). Namespace identifiers, like fully-qualified names are left-associative, which means `ns.ns2.task` is interpreted as `((ns.ns2).task)`, which means `ns.ns2` has to resolve to a namespace so that `.task` can be applied. If `ns2` is instead a task definition within `ns`, then the namespaced identifier `ns.ns2.task` is invalid.
+
+### Declarations
+
+```txt
+$declaration = $type $identifier ('=' $expression)?
+```
+
+A declaration reserves a name that can be referenced anywhere in the [scope](#appendix-b-wdl-namespaces-and-scopes) where it is declared. A declaration has a type, a name, and an optional initialization. Each declaration must be unique within its scope, and may not collide with a reserved WDL keyword (e.g. `workflow`, or `input`).
+
+Some examples of declarations:
+
+```wdl
+File x
+String y = "abc"
+Int i = 1 + 2
+Float pi = 3 + .14
+Map[String, String] m
+```
+
+A [task](#task-definition) or [workflow](#workflow-definition) may declare input parameters within its `input` section and output parameters within its `output` section. If a non-optional input declaration does not have an initialization, it is considered a "required" parameter, and its value must be provided by the user before the workflow or task may be run. Declarations may also appear in the body of a task or workflow. All non-input parameters must be initialized.
+
+```wdl
+task mytask {
+  input {
+    String s  # required input declaration
+    Int? i    # optional input declaration
+    Float f   # input delcaration with an initialization
+  }
+
+  Boolean b = true  # declaration in the task body
+
+  command <<< ... >>>
```

Comment:
This <<< ... >>> format is the first mention of it.  In fact, it's the first mention of the command section at all.  perhaps start with the command {...} format and then have the <<<...>>> as an optional one?

Reply:
Good point - I don't think it's necessary to refer to the command section at all here. I've just elided it.